jobbergate-cli/jobbergate_cli/config.py

```diff
@@ -93,22 +79,31 @@ def compute_extra_settings(self) -> Self:
         cache_dir = self.JOBBERGATE_CACHE_DIR
         cache_dir.mkdir(exist_ok=True, parents=True)
 
-        self.JOBBERGATE_APPLICATION_MODULE_PATH = cache_dir / constants.JOBBERGATE_APPLICATION_MODULE_FILE_NAME
-        self.JOBBERGATE_APPLICATION_CONFIG_PATH = cache_dir / constants.JOBBERGATE_APPLICATION_CONFIG_FILE_NAME
-
-        log_dir = cache_dir / "logs"
-        log_dir.mkdir(exist_ok=True, parents=True)
-        self.JOBBERGATE_LOG_PATH = log_dir / "jobbergate-cli.log"
+        return self
 
-        token_dir = cache_dir / "token"
+    @computed_field
+    def JOBBERGATE_USER_TOKEN_DIR(self) -> Path:
+        token_dir = self.JOBBERGATE_CACHE_DIR / "token"
         token_dir.mkdir(exist_ok=True, parents=True)
-        self.JOBBERGATE_USER_TOKEN_DIR = token_dir
-        self.JOBBERGATE_ACCESS_TOKEN_PATH = token_dir / "access.token"
-        self.JOBBERGATE_REFRESH_TOKEN_PATH = token_dir / "refresh.token"
+        return token_dir
 
-        self.JOBBERGATE_CLUSTER_LIST_PATH = cache_dir / "clusters.json"
+    @computed_field
+    def JOBBERGATE_LOG_PATH(self) -> Path:
+        log_file = self.JOBBERGATE_CACHE_DIR / "logs" / "jobbergate-cli.log"
+        log_file.parent.mkdir(exist_ok=True, parents=True)
+        return log_file
 
-        return self
+    @computed_field
+    def JOBBERGATE_APPLICATION_MODULE_PATH(self) -> Path:
+        return self.JOBBERGATE_CACHE_DIR / constants.JOBBERGATE_APPLICATION_MODULE_FILE_NAME
+
+    @computed_field
+    def JOBBERGATE_APPLICATION_CONFIG_PATH(self) -> Path:
+        return self.JOBBERGATE_CACHE_DIR / constants.JOBBERGATE_APPLICATION_CONFIG_FILE_NAME
+
+    @computed_field
+    def JOBBERGATE_CLUSTER_LIST_PATH(self) -> Path:
```

Comment:
I like what you did here in making these all `computed_fields`. It's a nice update.

Reply:
Thanks for noticing. All the mypy issues caused because they were marked as optional were bugging me, this make things more clear.